common/djangoapps/student/views.py

```diff
@@ -633,6 +633,17 @@ def dashboard(request):
         # otherwise, use the default language
         current_language = settings.LANGUAGE_DICT[settings.LANGUAGE_CODE]
 
+    poc_membership_triplets = []
+    if settings.FEATURES.get('PERSONAL_ONLINE_COURSES', False):
+        from pocs import ACTIVE_POC_KEY
+        from pocs.utils import get_poc_membership_triplets
```

Comment:
The intent is not to import unless the feature is active.  I am not married to this and would be happy to move the imports if that is more in keeping with the EdX style guide.